crates/sdk/src/api/value/mod.rs

```diff
@@ -248,7 +248,6 @@ transparent_wrapper!(
 impl_serialize_wrapper!(Number);
 
 impl Number {
-	#[doc(hidden)]
 	pub fn cource_into_i64(self) -> Option<i64> {
```

Comment:
If we really want to make this public we should fix this typo and add some documentation. This function also contains this branch

```rust
CoreNumber::Float(x) if x.fract() == x => Some(x as i64),
```

If I'm not mistaken that `fract` is actually supposed to be `trunc`, right?

```suggestion
	pub fn coerce_into_i64(self) -> Option<i64> {
```

Reply:
I don't think this function is used actually. It can probably be removed.